File: CodeForces/java/AntonAndPolyhedrons.java
import java.util.HashMap;
import java.util.Scanner;

public class AntonAndPolyhedrons {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        HashMap<String, Integer> list = new HashMap<>();
        list.put("Tetrahedron", 4);
        list.put("Cube", 6);
        list.put("Octahedron", 8);
        list.put("Dodecahedron", 12);
        list.put("Icosahedron", 20);

        int resul = 0;

        int n = s.nextInt();
        s.nextLine();

        for(int i=0; i<n; i++) {
            String input = s.nextLine();
            resul += list.get(input);
        }

        System.out.println(resul);
    }
}

File: CodeForces/c++/AntonAndPolyhedrons.cpp
#include <iostream>
#include <string>
#include <map>

int main()
{
	//We create the map
	std::map<std::string, int> list;
	list["Tetrahedron"]  =  4;
	list["Cube"]         =  6;
	list["Octahedron"]   =  8;
	list["Dodecahedron"] = 12;
	list["Icosahedron"]  = 20;

	int n;
		std::cin >> n;

	int result = 0;

	for (int i = 0; i < n; i++) {
		std::string figure;
			std::cin >> figure;
		result += list[figure]; //We access the value of the figure received and add it to the total result
	}
	std::cout << result;

	return 0;
}
